packages/scene-composer/src/components/StateManager.tsx

```diff
@@ -376,6 +373,11 @@ const StateManager: React.FC<SceneComposerInternalProps> = ({
         },
       });
     }
+
+    return () => {
```

Comment:
How did you test this? Could you please add an unit test for this change?

Reply:
added